outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "webapp_url" {
+  value = "https://${azurerm_app_service.main.default_site_hostname}"
+}
+
+output "azure_webhook_url" {
+  value = "https://${azurerm_app_service.main.site_credential[0].username}:${azurerm_app_service.main.site_credential[0].password}@${azurerm_app_service.main.name}.scm.azurewebsites.net/docker/hook"
```

Comment:
You may notice that the password here has already been obscured by Terraform in your pipeline, but even so I'd be inclined to add the `sensitive = true` tag to this output